rfcs/slider-component.md

```diff
@@ -0,0 +1,113 @@
+# Slider Component
+
+Represents control with sliding axis to choose a single value or range between min and max values.
+
+## Usage
+
+### Basic usage
+
+```javascript
+import * as React from 'react';
+import {Slider} from 'baseui/slider';
+
+export default () => <Slider range={[
+  {value: 0, label: '0, min'},
+  {value: 100, label: '100, max'}
+]} />;
+```
+
+### Advanced usage
+
+```javascript
+import * as React from 'react';
+import {StatefulSlider} from 'baseui/slider';
+
+export default () => {
+  return <StatefulSlider
+    range={[
+      {value: 0, label: '0, min'},
+      {value: 20, label: '20'},
+      {value: 40, label: '40'},
+      {value: 60, label: '60'},
+      {value: 80, label: '80'},
+      {value: 100, label: '100, max'}
+    ]}
+    initialState={{
+      value: [{value: 20, label: '20'}, {value: 40, label: '40'}]
+    }}
+    step={20}
```

Comment:
Below it says that step is ignored in the case of > 2 values passed in the `range` prop. Should probably remove it here to avoid a confusion.

Reply:
removed